services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -126,7 +128,25 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
         .recover { case e: Throwable => Left(VersioningException.HttpException(e.getMessage)) }
     }
 
-  override def getFiles(project: Project, path: Path)(implicit ec: ExecutionContext): AsyncResult[List[String]] = ???
+  override def getFiles(
```

Comment:
Maybe it will be better to place it near the getFile method

Reply:
Possibly, but it will cause more changes to the file and force the reviewer to do more work.